### PR TITLE
ESLint: Globally ignores "__" and "N_", fixes indents/formatting

### PR DESCRIPTION
**Pull request by AllenBW**
Indent fixes include: comma-dangle, indent,key-spacing, space-infix-ops
"__" and "N_" are in support of app translation

##report before this pr:
screen shot 2016-09-14 at 10 53 12 am

##report after this pr:
screen shot 2016-09-14 at 10 53 23 am

adds more notches against #121

**this one should only be merged after https://github.com/ManageIQ/manageiq-ui-self_service/pull/131**

### CODE REVIEW

File: client/app/components/blueprint-delete-modal/blueprint-delete-modal-service.factory.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
   function BlueprintDeleteFactory($modal) {
     var modalOpen = false;
     var modalBlueprint = {
-      showModal: showModal
+      showModal: showModal,
```
Comment (dtaylor113) [inline client/app/components/blueprint-delete-modal/blueprint-delete-modal-service.factory.js]: Aren't these commas unnecessary?!

Comment (AllenBW) [inline client/app/components/blueprint-delete-modal/blueprint-delete-modal-service.factory.js]: @dtaylor113 yeah great observation! soooooo we're going to start following the greater style guide, which says multiline trailing commas they are in style! 

https://github.com/ManageIQ/manageiq/issues/8781

it looks odd at first, but the pros out weight the cons on this one... (cons being no support in ie 8/9 but who cares about that :P )

File: .eslintrc.json [hunk 2]
```diff
@@ -38,7 +42,10 @@
       }
     ],
     "class-methods-use-this": "error",
-    "comma-dangle": "error",
+    "comma-dangle": [
+      "error",
+      "always-multiline"
```
Comment (dtaylor113) [inline .eslintrc.json]: Couldn't we change this to ""only-multiline", and avoid having to add all those unnecessary commas?!

Comment (dtaylor113) [inline .eslintrc.json]: ESLint doc states: "...IE8 ... and below will throw an error when it encounters trailing commas in JavaScript."  Does MiQ have to support IE8?  If so, then I'm ok with these changes.  -thanks

Comment (AllenBW) [inline .eslintrc.json]: uniformity across all, only-multiline will allow for dangling-comma to be absent, something we dont want..
again discussion about it in the comments https://github.com/ManageIQ/manageiq/issues/8781
@himdel can probably articulate the argument for a bit more eloquently than myself :)

**Comment by chriskacerguis** [comment]
@AllenBW looks like there are some conflicts, can you take a look?

**Comment by AllenBW** [comment]
yeah! expected, working it presently.  @chriskacerguis

**Pull request closed without merging**